<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Invoices</class>
 <widget class="QMainWindow" name="Invoices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>560</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bauhaus 93</family>
    <pointsize>72</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Invoices</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
border-radius: 20;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="datePicker1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border: 2px solid #6ad487;
border-radius: 10;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showInvoicesButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Broadway</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #6ad487;
border-radius: 20;
color: white; 
</string>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="datePicker2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border: 2px solid #6ad487;
border-radius: 10;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>941</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7; 
background-color: white;
border: 2px solid #6ad487;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newInvoiceButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>430</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara Light</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 20;
color: black; 
border: 2px solid #6ad487;</string>
    </property>
    <property name="text">
     <string>New invoice</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
